common-tools/clas-detector/src/main/java/org/jlab/detector/banks/RawOrderType.java

```diff
@@ -0,0 +1,41 @@
+package org.jlab.detector.banks;
+
+/**
+ * The second and third digits of ADC/TDC bank's order variable can be used
+ * to encode additional information (as long as the first digit is left
+ * unmanipulated), and this enum just defines the convention for those higher
+ * digits in production data.
+ *
+ * These order categories are all mutually exclusive, and denoising is
+ * categorized into levels, from 0 (least noisy) to 4 (most noisy).
+ * 
+ * @author baltzell
+ */
+public enum RawOrderType {
+
+    NOISE0          (  0),  // normal hits retained by denoising level-0
+    BGADDED         ( 10),  // hits added by background merging
```

Comment:
Just for clarity: should 10 be labeled as "hits added by background merging and retained by denoising"?

Reply:
no, that is in Gagik's granite, I was just trying to get the ball rolling here